hail/python/hail/experimental/datasets.json

```diff
@@ -4451,49 +4451,43 @@
         "reference_genome": "GRCh37",
         "url": {
           "aws": {
-            "us": "s3://hail-datasets-us-east-1/annotations/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht"
+            "us": "s3://hail-datasets-us-east-1/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht"
           },
           "gcp": {
-            "eu": "gs://hail-datasets-eu/annotations/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht",
-            "us": "gs://hail-datasets-us/annotations/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht"
+            "eu": "gs://hail-datasets-eu/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht",
+            "us": "gs://hail-datasets-us/LDSC_baselineLD_v2.2_ld_scores.GRCh37.ht"
           }
         },
         "version": "2.2"
       }
     ]
   },
   "ldsc_baselineLD_ldscores": {
-    "description": "LDSC Baseline-LD Model: contains 75 annotations (additional annotations included in later versions) including functional regions, histone marks, GERP scores, LD-related annotations of population genetics forces, MAF bins, and more.",
+    "description": "LDSC Baseline-LD Model: contains 75 annotations (additional annotations included in later versions) including functional regions, histone marks, GERP scores, LD-related annotations of population genetics forces, MAF bins, and more. Represented as a matrix table whose columns are annotations and entries are the ld scores. For a table representation, see ldsc_baselineLD_annotations.",
     "url": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6133304/",
     "versions": [
       {
         "reference_genome": "GRCh37",
         "url": {
-          "aws": {
-            "us": "gs://hail-datasets-eu/LDSC_baselineLD_v2.2_ld_scores.GRCh37.mt"
+          "us": {
```

Comment:
Looks good, only thing is `"us": {"us": ...}`  should be `"aws": {"us": ...}` here